lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -154,8 +154,16 @@ internal void CreateDefaultFolderStructure(AppParameters appParameters, string d
             File.WriteAllText(Path.Combine(driverFolderPath, _fileNames.GetBridgeExeConfigPath()), config);
 
             // generate .config file for Evaluator executable
-            File.WriteAllText(Path.Combine(driverFolderPath, _fileNames.GetGlobalFolderPath(), EvaluatorExecutable), 
-                DefaultDriverConfigurationFileContents);
+            var userDefinedEvaluatorConfigFileName = Path.Combine(JarFolder, EvaluatorExecutableConfig);
```

Comment:
How does a user provide this new config file? Here, we only test that it is next to the jar, right?

Reply:
We actually already have app.config file for Evaluator. It will be dropped in the binary folder together with evaluator.exe file. If users want to add more xml tags in the app.config file,, they can just modify that file without worrying about where to add it as a new file.

Reply:
How would a user know about this ability? As it stands, it seems as if the user has to *change* a file in the staging directory before submitting the job, right? Wouldn't it be easier to discover if we had a method in e.g. `JobRequestBuilder` where one could set that config file explicitly?

Reply:
Actually, maybe this method (also) belongs into `IAllocatedEvaluator`? We have a couple of processes that need `.config` files: The Driver, and each Evaluator. The Evaluators could potentially each need a different one.

Reply:
Theoretically each evaluator can have its own config file. And we should submit it with IEvaluatorRequest. But we don't support it yet. And it will involve much bigger changes. We can do it in a separate JIRA. 

This change is just to allow the user defined config file not to be overridden. It is needed by our applications immediately.